### DISCUSSION TITLE
Mods don't show up in mod manager

### DISCUSSION THREAD

**Discussion by elmiticosebas** [Raytwo/ARCropolis#519] [Issues]
ARCropolis ver 3.14.0 shows up in the start screen, but mods don't work. I've checked the mod manager and no mods show up, just the "?" icon. They are located in the correct folder (atmosphere/ultimate/mods) but I don't know what to do.
![image](https://user-images.githubusercontent.com/130189341/230686646-8694af71-0631-43d9-8273-348de8c8b873.png)
![image](https://user-images.githubusercontent.com/130189341/230686655-26d2f777-db2a-4560-a151-9323741c9f3e.png)

**Comment by redxtech** [comment]
What are the contents of each folder in `mods`?
